Assets/Scripts/Movement/Move.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+
+namespace Movement
+{
+    /// <summary>
+    /// Base movement class.
+    /// </summary>
+    public abstract class Move : MonoBehaviour
+    {
+        private void Update()
+        {
+            MoveOnUpdate();
```

Comment:
`MoveOnUpdate()` looks like an extra function to call. Every GameObject can "move" by manipulation `transform` object directly, including AI scripted events. Do you have a vision on where this would be used?

Reply:
Yeah, I guess this is an extra function call. Basically, I was trying to unify the `BasicMove` and `InputMove` classes under one general class, and this seemed like the best way to do it. Open to other suggestions though.

Reply:
I'd rather we don't do unification. `BasicMove` is just random script so I can get a cube moving. Seems unnecessary to optimize.

Reply:
Ok, removed.